docs/UserGuide.adoc

```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Get size of address book : `size`
+
+Output the size of address book +
+Format: `size`
```

Comment:
Keep it aligned with others (i.e. `Getting size of...`, `Outputs...`, full stop)